.github/workflows/stackhpc.yml

```diff
@@ -8,12 +8,7 @@ on:
   pull_request:
 jobs:
```

Comment:
Not directly related to this PR, but we should consider setting up jobs so that they don't run in forks: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository